db/migrate/20220209212113_create_books.rb

```diff
@@ -0,0 +1,15 @@
+class CreateBooks < ActiveRecord::Migration[6.1]
+  def change
+    create_table :books do |t|
+      t.string :title, null: false
+      t.string :author
+      t.string :description
+      t.string :isbn
+      t.string :cover_photo
+
+      t.timestamps
+      t.datetime :deleted_at
+      t.index :isbn, unique: true
```

Comment:
Is there a possibility that the board room will have several copies of the same book?